src/runtime-rs/crates/runtimes/virt_container/src/sandbox.rs

```diff
@@ -151,6 +190,16 @@ impl Sandbox for VirtSandbox {
         self.hypervisor.start_vm(10_000).await.context("start vm")?;
         info!(sl!(), "start vm");
 
+        // execute pre-start hook functions, including Prestart Hooks and CreateRuntime Hooks
+        let (prestart_hooks, create_runtime_hooks) = match spec.hooks.as_ref() {
+            Some(hooks) => (hooks.prestart.clone(), hooks.create_runtime.clone()),
+            None => (Vec::new(), Vec::new()),
+        };
+        self.execute_prestart_hooks(&prestart_hooks, &create_runtime_hooks, state)
+            .await?;
+
+        // TODO: if prestart_hooks is empty, rescan the network endpoints
```

Comment:
Why leave this comment?

Reply:
Sorry, it should be "is not". I will fix it.
This comment is related to https://github.com/kata-containers/kata-containers/pull/5972. 
As networks may be configured through `Prestart Hook`, so we should rescan the network if `prestart_hook` is not empty. Hotplug network endpoints has not supported yet and the rescan operation relies on that capability,  so I leave this comment as a TODO action.